.ci/install_linux_cmake.sh

```diff
@@ -60,7 +57,3 @@ fi
 if [ $BUILD_NAME = DOCS ]; then
   $SUDO apt-get -qq -y install doxygen
 fi
-
-if [ $BUILD_NAME = TRUSTY_FULL_DEBUG ]; then
-  sudo apt-get install -y clang-format-6.0
```

Comment:
Do we need to replace this with clang-format-10.0 (as you did in `install_linux_catkin`?)

Reply:
Nope, clang-format-10 is only needed for the FOCAL_CATKIN_DEBUG_CODECOV (we only need to test formatting in one job), so the `cmake` installation doesn't need it.